spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -205,7 +205,10 @@
       expect(@analytics).to receive(:track_event).
         with('Email and Password Authentication', analytics_hash)
 
-      post :create, params: { user: { email: user.email.upcase, password: user.password } }
+      expect_any_instance_of(IrsAttemptsApi::Tracker).to receive(:email_and_password_auth).
+        with(email: user.email, success: true)
+
+      post :create, params: { user: { email: user.email, password: user.password } }
```

Comment:
I removed the `.upcase` that was originally here because it was apparently unnecessary. The expectation I added obviously had to match, but I couldn't figure out why we were upper-casing the email.